### PR TITLE
Added the option to set menu links as not active to hide them without…

### PR DESCRIPTION
**Pull request by spitfire305**
… deleting them from the db.

### CODE REVIEW

File: resources/views/menu_links/form/_entry.blade.php [hunk 1]
```diff
@@ -110,6 +110,14 @@
             @endif
         </div>
     </div>
+    <div class="col-md-6">
+        {!! Form::hidden('is_active', 0) !!}
+        <label>
+            {!! Form::checkbox('is_active', 1, $model->is_active) !!}
+            {!! __('menu_links.fields.active') !!}
+            <i class="fa-solid fa-question-circle" data-toggle="tooltip" title="{{ __('menu_links.helpers.active') }}"></i>
```
Comment (ilestis) [inline resources/views/menu_links/form/_entry.blade.php:118]: Missing the helper block for 40% of users on mobile devices :) the question-circle needs to be hidden on mobile devices too.

**Review by ilestis — CHANGES_REQUESTED** [review]

File: resources/views/menu_links/form/_entry.blade.php [hunk 2]
```diff
@@ -113,10 +113,14 @@
...
         {!! Form::hidden('is_active', 0) !!}
         <label>
-            {!! Form::checkbox('is_active', 1, $model->is_active) !!}
+            {!! Form::checkbox('is_active', 1, isset($model) ? $model->is_active : 1) !!}
             {!! __('menu_links.fields.active') !!}
-            <i class="fa-solid fa-question-circle" data-toggle="tooltip" title="{{ __('menu_links.helpers.active') }}"></i>
+            <i class="fa-solid fa-question-circle" data-toggle="tooltip" data-html="true" title="{{ __('menu_links.helpers.active') }}"></i>
         </label>
+        <div class="help-block visible-xs visible-sm">
+            {!! __('entities/notes.hints.reorder', ['icon' => '<i class="fa-solid fa-cog"></i>']) !!}
```
Comment (ilestis) [inline resources/views/menu_links/form/_entry.blade.php:121]: Wrong message shown on mobile

File: resources/views/menu_links/form/_entry.blade.php [hunk 3]
```diff
@@ -113,10 +113,14 @@
     <div class="col-md-6">
         {!! Form::hidden('is_active', 0) !!}
         <label>
-            {!! Form::checkbox('is_active', 1, $model->is_active) !!}
+            {!! Form::checkbox('is_active', 1, isset($model) ? $model->is_active : 1) !!}
             {!! __('menu_links.fields.active') !!}
-            <i class="fa-solid fa-question-circle" data-toggle="tooltip" title="{{ __('menu_links.helpers.active') }}"></i>
+            <i class="fa-solid fa-question-circle" data-toggle="tooltip" data-html="true" title="{{ __('menu_links.helpers.active') }}"></i>
```
Comment (ilestis) [inline resources/views/menu_links/form/_entry.blade.php:118]: Not sure that data-html="true" is needed, but the icon is still visible on small screens

**Review by ilestis — CHANGES_REQUESTED** [review]

File: resources/views/menu_links/form/_entry.blade.php (hunk 2 above)
Comment (spitfire305) [reply resources/views/menu_links/form/_entry.blade.php]: Oops

**Review by ilestis — APPROVED** [review]
🥳

**Pull request merged**